playground.sh

```diff
@@ -18,44 +18,104 @@
 # under the License.
 #
 
-set -x
-
 playground_dir="$(dirname "${BASH_SOURCE-$0}")"
 playground_dir="$(
-  cd "${playground_dir}" >/dev/null
+  cd "${playground_dir}" >/dev/null || exit 1
   pwd
 )"
 
+requiredDiskSpaceGB=25
+requiredRamGB=8
+requiredCpuCores=2
+requiredPorts=(8090 9001 3307 19000 19083 60070 13306 15342 18080 18888 19090 13000)
+
 testDocker() {
-  echo "INFO: Testing Docker environment by running hello-world..."
-  docker run --pull always hello-world >/dev/null 2>&1
+  echo "[INFO] Testing Docker environment by running hello-world..."
+  # Use always to test network connection
+  docker run --rm --pull always hello-world:latest >/dev/null 2>&1
```

Comment:
Please specify `hello-world` fixed tag, do not use `latest`

Reply:
I use the latest tag because `docker --pull always` ensures that an image is always downloaded from the network and updates the local image. Therefore, using any specific tag would yield the same result. However, I can still switch to a more stable version for consistency.

Reply:
Got it.